docs/source/getting-started/toc.rst

```diff
@@ -1,31 +1,30 @@
-==================
+===============
 Getting Started
-==================
+===============
 
-This section of the documentation is a no work experience required walkthrough of the Augur project. By the end, you’ll hopefully have a fully functioning local installation of Augur ready to collect data.
+This section of the documentation is a no work experience required walkthrough of the Augur project. By the end, you'll hopefully have a fully functioning local installation of Augur ready to collect data.
 
-If you want to get started as fast as possible, we have `Docker images <../docker/toc.html>`_; however, if you’re looking to use Augur for long-term data collection or if you want to install it for development, you’ll need to follow this walkthrough.
+Choose Your Installation Method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Augur offers two primary installation approaches:
 
-.. note::
-
-    We currently officially support the local installation of Augur from source on macOS, Ubuntu, and Fedora (but most UNIX-like systems will probably work with a few tweaks). We recommend either using the Docker images or setting up a virtual machine with a supported operating system installed if you are using Windows.
+**Docker (Recommended for Quickstart)**: The fastest way to get Augur up and running with minimal setup. Ideal for trying out Augur, demonstrations, or light installations (~< 2,000 repos).
 
-To install from source, we'll need to do a few things:
+**Manual Installation (Recommended for Long-term Use)**: For production deployments, long-term data collection, or development work. We officially support macOS, Ubuntu, and Fedora.
 
-1. Setup a PostgreSQL instance to store the data collected by Augur
-2. Install and configure Augur's application server
-3. Install and configure Augur's data collection workers
+.. note::
 
-The next section will start with a database setup, and then you can continue with the following steps given below.
+    We recommend either using the Docker images or setting up a virtual machine with a supported operating system installed if you are using Windows.
```

Comment:
docker installs on windows will basically already run in a virtual machine behind the scenes because containers rely on features of the linux kernel